src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/wikipage/WikiArticleHomePage.java

```diff
@@ -6,7 +6,7 @@
 
 public class WikiArticleHomePage extends WikiBasePageObject {
 
-  private By wikiHomePageSpecificElement = By.cssSelector("#HOME_TOP_LEADERBOARD");
+  private By wikiHomePageSpecificElement = By.cssSelector(".mainpage");
```

Comment:
I want to suggest running some other tests which use it since I know AdEng tests passed but let's make sure other selenium tests passes. The worry comes from the fact that the selector used to be a unique id `#HOME_TOP_LEADERBOARD` and now it's going to be changed to a class `.mainpage`. The adeng tests may not be used `wikiHomePageSpecificElement` and passed whereas the `By.cssSelector()` may now return a collection and cause failures in some other tests. What do you think about this concern?